#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_mimetype.dpatch
##
## DP: update desktop entry

@DPATCH@
--- a/extra/alsaplayer.desktop
+++ b/extra/alsaplayer.desktop
@@ -4,9 +4,9 @@
 Type=Application
 Comment=Versatile audio player for the Audio Linux Sound Architecture
 Comment[fr]=lecteur audio versatile for the Audio Linux Sound Architecture
-Exec=alsaplayer
+Exec=alsaplayer %U
 TryExec=alsaplayer
 Icon=alsaplayer
-MimeType=audio/x-scpls;audio/x-mpegurl;audio/mpegurl;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;audio/ogg+vorbis;audio/ogg;application/flac;
+MimeType=audio/x-scpls;audio/x-mpegurl;audio/mpegurl;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;audio/ogg+vorbis;audio/ogg;audio/flac;
 Categories=AudioVideo;Audio;Player;
 StartupNotify=true
